data/test-cases/plugins/jira/miscellaneous/server/Tooltip_over_link.md

```diff
@@ -0,0 +1,62 @@
+---
+# (Required) Ensure all values are filled up
+name: "Tooltip over link is showing correct ticket information on the server instance."
```

Comment:
Please fix the above comments here also, and also add the Jira server or Jira cloud wherever specifying any task which would be completed on Jira.

Reply:
Where ever i have edited a issue and deleted a issue i have added the respective cloud and server.